app/views/layouts/_footer.html.haml

```diff
@@ -1,32 +1,9 @@
-.footer-sponsors.l-align-center
-  %ul.list-inline.footer-sponsors-list
-    %li.footer-sponsor-list
-      = link_to 'http://www.jollygoodcode.com/', target: '_blank' do
-        %h5.l-align-center
-          %i= t('footer.supported_by')
-
-        .sponsor-jolly-good-code
-          %span.l-text-hidden Jolly Good Code
-
-    %li.footer-sponsor-list
-      = link_to 'http://www.discourse.org/', target: '_blank' do
-        %h5.l-align-center
-          %i= t('footer.domain_and_community')
-
-        .sponsor-discourse
-          %span.l-text-hidden Discourse
-
-    %li.footer-sponsor-list
-      = link_to 'http://www.rubytune.com/', target: '_blank' do
-        %h5.l-align-center
-          %i= t('footer.bare_metal_server')
-
-        .sponsor-ruby-tune
-          %span.l-text-hidden RubyTune
```

Comment:
I think we should keep this. Since it appears at the footer, we give the sponsors a higher chance at exposure. My intention for the dedicated sponsors page was to help provide more information about each sponsor.

@SamSaffron Any thoughts?